### ISSUE TITLE
Tracking Code Redirect Error

### ISSUE THREAD

**Issue by VR51** [mautic/mautic#4416]
I sent an email to just over 8000 list members as part of a subscriber reconfirmation procedure. I now have an interesting little problem caused by the auto generated tracking code query string. All users who click the link get a 403 Forbidden message.

Take a look at this query string 

`example.com/r/8b6b27ba5a5cb2c35377fcf8e?ct=YTo1OntzOjY6InNvdXJjZSI7YToyOntpOjA7czo1OiJlbWFpbCI7aToxO2k6MTt9czo1OiJlbWFpbCI7aToxO3M6NDoic3RhdCI7czoxMzoiNTk2NTY0ZjBkMTA2ZiI7czo0OiJsZWFkIjtzOjQ6IjQ2MzMiO3M6NzoiY2hhbm5lbCI7YToxOntzOjU6ImVtYWlsIjtpOjE7fX0**%3D**`

Notice the highlighted part **%3D**.

I thought I could quickly resolve the problem by setting up a wildcard redirect on the domain so that every visitor gets sent off to a Mailchimp sign up form. I was wrong.

I've spent the last few hours and a few hours here in there over the past 24 hours coming back to the problem of getting that URL to behave properly but not one solution I tried worked whether that be RedirectMatch, RewriteCond/RewriteRule combinations, removal of all other redirect rules.... Nope. Nothing worked to get that URL to either drop the query string, redirect to another domain or, as mentioned already, getting the whole domain to redirect to somewhere else for every request.

I should have thought of testing this first. The URL looks quite long so I wondered what would happen if I removed characters of the URL one-by-one. Sure enough, I only needed to delete %3D from the URL and the redirects work lovely.

I can move that %3D to different parts of the tracking code and the problem still remains.

%3D is the code for '='. My web server does not like it one bit. It is not alone in that.

Can you please adjust the tracking codes to ensure they do not include character combinations that some web servers will throw tantrums over.

Thank you.

**Comment by escopecz** [comment]
See https://stackoverflow.com/questions/4016067/whats-a-3d-doing-in-this-html for the 3D part. It's normal email encoding you can find in the email source. But it's decoded by the email clients. My guess is that 403 is some server misconfiguration. Where are you redirected when you actually click the email link?

**Comment by VR51** [comment]
The moment I posted this I realised what I'd done. The fault was caused by mod_security. Had forgotten I'd installed new mod_sec rules to the server yesterday while reconfiguring it. I'm now going to sit in front of a mirror and tell myself off.

Thanks for your quick reply.

On another note, I ran the Mautic software update at some point during the last 24 hours. The update failed on the database component. Is there a way to quickly force the DB to upgrade?
